Experiment 'pythonaa' is active
Experiment 'pythonTensorboardExperiment' is active
Experiment 'PythonPyTorchProfiler' is active
Experiment 'pythonDeprecatePythonPath' is active
> conda info --json
> /usr/bin/python3 ~/.vscode/extensions/ms-python.python-2021.12.1559732655/pythonFiles/interpreterInfo.py
Python interpreter path: /usr/bin/python3
[ERROR 2021-11-19 1:18:0.845]: readdir() failed for "/home/nighttsu/.cache/dconf" (Error: EACCES: permission denied, scandir '/home/nighttsu/.cache/dconf')
[ERROR 2021-11-19 1:18:0.911]: readdir() failed for "/home/nighttsu/.cache/doc" (Error: EACCES: permission denied, scandir '/home/nighttsu/.cache/doc')
[ERROR 2021-11-19 1:18:1.364]: readdir() failed for "/home/nighttsu/.cache/nvim" (Error: EACCES: permission denied, scandir '/home/nighttsu/.cache/nvim')
> /usr/bin/python ~/.vscode/extensions/ms-python.python-2021.12.1559732655/pythonFiles/interpreterInfo.py
> /usr/include/python3.6 ~/.vscode/extensions/ms-python.python-2021.12.1559732655/pythonFiles/interpreterInfo.py
[ERROR 2021-11-19 1:18:2.746]: readdir() failed for "/home/nighttsu/.config/ristretto" (Error: EACCES: permission denied, scandir '/home/nighttsu/.config/ristretto')
[ERROR 2021-11-19 1:18:2.830]: readdir() failed for "/home/nighttsu/.dbus" (Error: EACCES: permission denied, scandir '/home/nighttsu/.dbus')
[ERROR 2021-11-19 1:18:2.896]: readdir() failed for "/home/nighttsu/.gvfs" (Error: EACCES: permission denied, scandir '/home/nighttsu/.gvfs')
Starting Pylance language server.
> conda --version
